tests/devices/unit_tests/test_shutter.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+from ophyd_async.core import DeviceCollector
+
+from dodal.devices.shutter import OpenState, Shutter
+
+
+@pytest.fixture
+async def sim_shutter():
+    async with DeviceCollector(sim=True):
+        sim_shutter: Shutter = Shutter(
+            prefix="sim_shutter",
+            name="shutter",
+        )
+    return sim_shutter
+
+
+@pytest.mark.parametrize(
+    "state",
+    [
+        (OpenState.OPEN),
+        (OpenState.CLOSE),
+    ],
+)
+async def test_set_opens_and_closes_shutter(state: OpenState, sim_shutter: Shutter):
+    status = sim_shutter.set(state)
+    assert not status.done
+    reading = await sim_shutter.read()
+
+    shutter_position = reading.get("shutter-position", {})
+    assert (
+        shutter_position["value"] in OpenState
+    ), f"Unexpected value: {shutter_position['value']}"
```

Comment:
Should assert that the state is the expected state

Reply:
that is asserted some lines later, should we skip dealing with the timestamp?